drivers/display/ssd16xx.c

```diff
@@ -580,7 +583,7 @@ static int ssd16xx_init(struct device *dev)
 	}
 
 	gpio_pin_configure(driver->dc, SSD16XX_DC_PIN,
-			   GPIO_DIR_OUT);
+			   GPIO_OUTPUT | SSD16XX_DC_FLAGS);
```

Comment:
`GPIO_OUTPUT_INACTIVE`?  Just for clarity/consistency?

Reply:
okay, but DC will always be set/reset in ssd16xx_write_cmd(). btw, I forgot to change few lines, fixed now.